candy-machine/program/src/constants.rs

```diff
@@ -22,7 +22,7 @@ pub const CONFIG_ARRAY_START: usize = 8 + // key
     8 + // price
     8 + // items available
     9 + // go live
-    10 + // end settings
+    16 + // end settings
```

Comment:
confused why this was set to 10, checking out the space `std::mem::size_of::<EndSettings>` returns 16

Reply:
Does this check the already deserialized size though? I think the size would be different after it is serialized in borsh, no? Almost all our types show different sizes than the ones we calculated in the constants.rs. I feel like candy machines with all the settings active would have broken by now (which we have tested and they shouldn't break?)

Reply:
reverted back to 10 since you are right config size offset change would not be backwards compatible with existing reads without a v3